SPARQLWrapper/Wrapper.py

```diff
@@ -1310,8 +1327,12 @@ def __get_results_width(self, results, minWidth=2):
                 index += 1
         return width
 
-    def __get_prettyprint_string_sparql_var_result(self, result):
+    def __get_prettyprint_string_sparql_var_result(self, result: Dict[str, str]) -> str:
+        if "value" not in result:
+            return ""
         value = result["value"]
+        if not isinstance(value, str):
+            raise TypeError(type(value))
```

Comment:
I would say this is somewhat unnecessary, if a user wants to pass in a dict with values that are not strings even though the type annotations say they should be strings I think it is fine if they bear the consequences. 

Also it is not clear where the limiting principle for an approach of runtime type checking arguments.

In this case it probably does not matter much, but given that this is a bit of an outlier I would rather remove it, but will approve with it also.

```suggestion
```